client/src/pages/Login.jsx

```diff
@@ -41,13 +41,11 @@ function Login() {
         </p>
       </Form>
 
-      <article>
-        <img
-          src="../src/assets/images/mockup.png"
-          alt="Mockups de l'application"
-          className="imgMockup"
-        />
-      </article>
+      <img
+        src="../src/assets/images/mockup.png"
```

Comment:
You can add the image in a "const" and then use it here

Reply:
done